### PR TITLE
ADCM-1322 Added tests for config. 

### PR DESCRIPTION
**Pull request by a-alferov**

### CODE REVIEW

File: tests/functional/test_config.py [hunk 1]
```diff
@@ -0,0 +1,488 @@
...
+    return processing_data(sdk_client_ms, request, 'not_required_without_default_sent_empty_value')
+
+
+@fixture_parametrized_by_data_subdirs(
+    __file__, 'not_required', 'without_default', 'sent_null_value', scope='module')
+def nr_wod_nv(sdk_client_ms: ADCMClient, request):
+    return processing_data(sdk_client_ms, request, 'not_required_without_default_sent_null_value')
+
+
+def test_not_required_without_default_sent_correct_value(nr_wod_cv):
```
Comment (acmnu) [inline tests/functional/test_config.py:418]: Test scenario should be commented here in help message form.

File: tests/functional/test_config.py [hunk 2]
```diff
@@ -0,0 +1,488 @@
...
+    assert_config_type(*r_wd_ev, True, True, 'empty_value')
+
+
+def test_required_with_default_sent_null_value(r_wd_nv):
+    assert_config_type(*r_wd_nv, True, True, 'null_value')
+
+
+@fixture_parametrized_by_data_subdirs(
+    __file__, 'required', 'without_default', 'sent_correct_value', scope='module')
+def r_wod_cv(sdk_client_ms: ADCMClient, request):
```
Comment (acmnu) [inline tests/functional/test_config.py:462]: help message needed. Func name too short to understand what is going on here.

Comment (a-alferov) [reply tests/functional/test_config.py:462]: name is formed from parameters "fixture_parametrized_by_data_subdirs(
    __file__, 'required', 'without_default', 'sent_correct_value', scope='module')" 
'required', 'without_default', 'sent_correct_value' - r_wod_cv

File: tests/functional/test_config.py (hunk 1 above)
Comment (a-alferov) [reply tests/functional/test_config.py:493]: Ok

**Review by acmnu — APPROVED** [review]

**Pull request merged**